src/aashay/ConditionalStatement/Vowel_Switch.java

```diff
@@ -0,0 +1,56 @@
+package aashay.ConditionalStatement;
+/*
+ * Assignment - 13 : 7th April'2021
+
+WAP to evaluate whether given character is vowel or not using switch case.
+sample input1 : d
+sample output1 : d is not a vowel
+  
+sample input2 : I
+sample output2 : I is a vowel
+
+sample input3 : i
+sample output3 : i is a vowel
+ */
+
+public class Vowel_Switch {
+	
+	void vowel(String ch) {
+		switch(ch) {
```

Comment:
what if someone pass capitals char?
we need to handle that condition as well

take reference of below code

Pravin. - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/496/files
Neha R- https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/497/files
Aparana - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/533/files
Paras - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/526/files